tests/system/feature_store/test_spark_engine.py

```diff
@@ -216,3 +219,62 @@ def test_schedule_on_filtered_by_time(self, partitioned):
         resp = fs.get_offline_features(vec)
         assert len(resp.to_dataframe() == 4)
         assert "uri" not in resp.to_dataframe() and "katya" not in resp.to_dataframe()
+
+    def test_aggregations(self):
+        name = f"measurements_{uuid.uuid4()}"
+
+        test_base_time = datetime.fromisoformat("2020-07-21T21:40:00+00:00")
+
+        df = pd.DataFrame(
+            {
+                "time": [
+                    test_base_time,
+                    test_base_time + pd.Timedelta(minutes=1),
+                    test_base_time + pd.Timedelta(minutes=2),
+                    test_base_time + pd.Timedelta(minutes=3),
+                    test_base_time + pd.Timedelta(minutes=4),
+                ],
+                "first_name": ["moshe", "yosi", "yosi", "moshe", "yosi"],
+                "last_name": ["cohen", "levi", "levi", "cohen", "levi"],
+                "bid": [2000, 10, 11, 12, 16],
+            }
+        )
+
+        path = "v3io:///bigdata/test_aggregations.parquet"
+        fsys = fsspec.filesystem(v3iofs.fs.V3ioFS.protocol)
+        df.to_parquet(path=path, filesystem=fsys)
+
+        source = ParquetSource("myparquet", path=path, time_field="time")
+
+        data_set = fs.FeatureSet(
```

Comment:
I think that we should add a storey engine feature set and verify that the schemas are similar.

Reply:
I was going to add it, but it feels kind of wrong to put a storey test inside of a spark test...

Reply:
It's simple ingest. All our tests somehow test storey...

Reply:
Because they test the storey engine, not the spark engine... But sure, I can do it... Just seems weird...